script.py

```diff
@@ -1,13 +1,38 @@
 import os
-from datetime import datetime
+import requests
 
 # Get input from environment variables
+website = os.getenv("INPUT_WEBSITE", "")
+test_name = os.getenv("INPUT_TEST_NAME", "")
+cj_token = os.getenv("INPUT_CJ_TOKEN", "")
 instructions = os.getenv("INPUT_INSTRUCTIONS", "")
 
-# Perform some processing
-output = f"Input received: {instructions}. Current time is {datetime.now()}."
+def run():
+    if not (website and test_name and cj_token and instructions):
+        return "Failed: All required fields were not provided."
+
+    response = requests.post(
+        "https://cj-backend.foreai.co/test-case/",
+        headers={
+            "Authorization": f"Bearer {cj_token}"},
+        json = {
+            "website": website,
+            "name": test_name,
+            "description": instructions,
```

Comment:
unrelated note: 'description' is a bad API name for the instructions - we should update it in v2 of the API

Reply:
Noted. Maybe we should change the `website` field as well.